.github/workflows/unit_test.yml

```diff
@@ -66,3 +66,14 @@ jobs:
       run: if [[ ${{ matrix.python-version }} == 3.9 ]]; then bash <(curl -s https://codecov.io/bash); fi
       env:
         CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Shell check mycroft tools
+      uses: ludeeus/action-shellcheck@master
```

Comment:
Can we set the action-shellcheck to a specific commit rather always using master?

This is a [recommended practice by Github](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_iduses) as it removes the ability for the action author or some other malicious actor to add unexpected code to that repo.

`actions/checkout@v2` is fine as it's an official Github Action, but this one I'd say is more vulnerable.

Reply:
good suggestion, I'll fix that tonight

Reply:
Updated locking it to the current commit on master.